controllers/operator_controller.go

```diff
@@ -314,6 +314,16 @@ func verifyBDStatus(dep *rukpakv1alpha1.BundleDeployment) (metav1.ConditionStatu
 
 // mapBDStatusToReadyCondition returns the operator object's "TypeReady" condition based on the bundle deployment statuses.
 func mapBDStatusToReadyCondition(existingBD *rukpakv1alpha1.BundleDeployment, observedGeneration int64) metav1.Condition {
+	// if BundleDeployment status is stale, return an unknown condition.
+	if isBundleDepStale(existingBD) {
+		return metav1.Condition{
+			Type:               operatorsv1alpha1.TypeReady,
+			Status:             metav1.ConditionUnknown,
+			Reason:             operatorsv1alpha1.ReasonInstallationStatusUnknown,
+			Message:            fmt.Sprintf("waiting for bundleDeployment %q status to be updated. BundleDeployment conditions out of date.", existingBD.Name),
```

Comment:
nit: use consistent case on BundleDeployment? I realize the first use is in the middle of a sentence and the second is at the beginning, but we aren't capitalizing "Waiting" in the first sentence.

My suggestion is that if we're talking about a kind, capitalize it always, just like its defined in the API.

```suggestion
			Message:            fmt.Sprintf("waiting for BundleDeployment %q status to be updated. BundleDeployment conditions out of date.", existingBD.Name),
```